src/app/components/Upload.svelte

```diff
@@ -110,10 +110,11 @@
 
 <div
   class="upload-zone"
-  class:drag-over={onDraggedOver}
+  class:drag-over={dragEnterCounter > 0}
```

Comment:
This is fascinating! Could you please add a comment about this behaviour somewhere in the code with a link to the fact that Chrome is bad at dragover?